packages/origin-ui-core/src/components/bundles/CertificateGroup.tsx

```diff
@@ -106,7 +106,7 @@ export const CertificateGroup = (props: IOwnProps) => {
                                                 <div>
                                                     <Typography>{type}</Typography>
                                                     <Typography>
-                                                        {PowerFormatter.format(
+                                                        {EnergyFormatter.format(
                                                             energy.toNumber(),
```

Comment:
Please don't cast `energy` to number. This might cause issues with big numbers. Forward just `energy` to EnergyFormatter.